zvariant_utils/src/case.rs

```diff
@@ -8,7 +8,7 @@ pub fn pascal_or_camel_case(s: &str, is_pascal_case: bool) -> String {
     let mut capitalize = is_pascal_case;
     let mut first = true;
     for ch in s.chars() {
-        if ch == '_' {
+        if ch == '_' || ch == '-' {
```

Comment:
This makes the function, also handle `kebab-case` so docs should be updated.

Reply:
You are right I will update this.